### PR TITLE
Added flow type check ReactDOMStringRenderer.

### PR DESCRIPTION
**Pull request by abhaynikam**
Trying to add flow for the first time so starting with small code. Related to #11328

Please feel free to close PR if this is not required.

### CODE REVIEW

File: packages/react-dom/src/server/ReactDOMStringRenderer.js [hunk 1]
```diff
@@ -12,7 +14,7 @@ import ReactPartialRenderer from './ReactPartialRenderer';
  * server.
  * See https://reactjs.org/docs/react-dom-server.html#rendertostring
  */
-export function renderToString(element) {
+export function renderToString(element: Element) {
```
Comment (gaearon) [inline packages/react-dom/src/server/ReactDOMStringRenderer.js]: This is a DOM element. It's not related to React elements (which are the inputs to this function).

Comment (abhaynikam) [reply packages/react-dom/src/server/ReactDOMStringRenderer.js]: Oh okay. So flow type here should be `any`. Right?

Comment (gaearon) [reply packages/react-dom/src/server/ReactDOMStringRenderer.js]: Kind of, but I don’t see how this helps us in general. It’s entry point so its type doesn’t matter in the internal flow check.

**Comment by gaearon** [comment]
Thanks—not taking because the type is wrong and because typing entry point doesn’t increase coverage in practice. We’d appreciate PRs for the things we do use though!

**Comment by abhaynikam** [comment]
@gaearon : Thanks for taking a look. Do we have the list of the component which needs type check?

**Pull request closed without merging**